package.json

```diff
@@ -11,7 +11,18 @@
   "configSchema": {
     "executablePath": {
       "type": "string",
-      "default": "codeclimate"
+      "default": "/usr/local/bin/codeclimate",
```

Comment:
Exactly! Let the OS handle path resolution for us.

Although I notice you still have the executable as `/bin/bash`, is that necessary or can we just pass `codeclimate` in directly?

(I have yet to get this working on Windows as there is no wrapper script that I've found, so I can't test this locally)